seacrowd/sea_datasets/vsolscsum/vsolscsum.py

```diff
@@ -0,0 +1,244 @@
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+        """Returns SplitGenerators."""
+
+        data_path = Path(dl_manager.download_and_extract(_URLS[_DATASETNAME]))
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "filepath": data_path,
+                    "split": "train",
+                },
+            )
+        ]
+
+    def _generate_examples(self, filepath: Path, split: str) -> Tuple[int, Dict]:
+        """Yields examples as (key, example) tuples."""
+
+        with open(filepath, "r", encoding="utf-8") as file:
+            xml_content = file.read()
+
+        root = ET.fromstring(xml_content)
+
+        def extract_data_from_xml(root):
+            data = []
+
+            for post in root.findall(".//post"):
+                post_id = post.get("id")
+                title = post.find("title").text
+                summary_sentences = [sentence.find("content").text for sentence in post.find(".//summary").find("sentences").findall("sentence")]
+                document_sentences = [sentence.find("content").text for sentence in post.find(".//document").find("sentences").findall("sentence")]
+                comment_sentences = [sentence.find("content").text for sentence in post.find(".//comments").find(".//comment").find("sentences").findall("sentence")]
+
+                summary_text = " ".join(summary_sentences)
+                document_text = " ".join(document_sentences)
+                comment_text = " ".join(comment_sentences)
+
+                data.append(
+                    {
+                        "post_id": post_id,
+                        "title": title,
+                        "summary": summary_text,
+                        "document_and_comment": f"{document_text} | {comment_text}",
+                    }
+                )
+
+            return data
+
+        extracted_data = extract_data_from_xml(root)
+        df = pd.DataFrame(extracted_data)
+
+        for index, row in df.iterrows():
+
+            if self.config.schema == "source":
+                example = row.to_dict()
+
+            elif self.config.schema == "seacrowd_t2t":
+
+                example = {
+                    "id": str(row["post_id"]),
+                    "text_1": str(row["summary"]),
+                    "text_2": str(row["document_and_comment"]),
+                    "text_1_name": "summary",
+                    "text_2_name": "document_and_comment",
+                }
+
+            yield index, example
+
+
+# This template is based on the following template from the datasets package:
+# https://github.com/huggingface/datasets/blob/master/templates/new_dataset_script.py
```

Comment:
Let's remove this line

Reply:
Solved on https://github.com/SEACrowd/seacrowd-datahub/pull/609/commits/60c52750e9f190d1254e40b107e0235072619df0.